package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "type": "module",
   "scripts": {
-    "start": "nodemon ./src/server.js",
+    "start": "nodemon -e \"*\" ./src/server.js",
```

Comment:
Added wildcard extension so both .js and .yaml files trigger a server reload.

Reply:
This is handy. I think originally there was a problem with getting stuck in an infinite loop because every start up would generate a new yaml file and trigger a restart, and so on and so on.... Since we have now changed to an in memory specification this will work well!